tapiro/core/src/main/scala/io/buildo/tapiro/Http4sMeta.scala

```diff
@@ -44,17 +45,10 @@ object Http4sMeta {
     q"Router($route -> NonEmptyList($first, List(..$rest)).reduceK)"
```

Comment:
not related to this PR, but this can be:

```suggestion
    q"Router($route -> NonEmptyList.of(first, rest: _*).reduceK)"
```